src/cloud-api-adaptor/docs/SecureComms.md

```diff
@@ -27,6 +27,34 @@ Once the "Kubernetes Phase" SSH channel is established, Secure Comms connects th
 
 See [Secure Comms Architecture Slides](./SecureComms.pdf) for more details.
 
+## Setup for for testing without Trustee (and for non-CoCo peerpods)
+
+### Deploy CAA
+Use any of the option for installing CAA depending on the cloud driver used.
+
+
+### Activate Secure-Comms feature from CAA side
+Activate Secure-Comms from CAA side by changing the `SECURE_COMMS` parameter of the `peer-pods-cm` configMap in the `confidential-containers-system` namespace to `"true"`.  
+
+```sh
+kubectl -n confidential-containers-system  get cm peer-pods-cm  -o yaml | sed "s/SECURE_COMMS: \"false\"/SECURE_COMMS: \"true\"/"|kubectl apply -f -
+```
+
+You may also include additional Inbounds and Outbounds configurations to the Adaptor side using the `SECURE_COMMS_INBOUNDS` and `SECURE_COMMS_OUTBOUNDS` config points.
+You may also add Inbounds and Outbounds configurations to the Forwarder side using the `SECURE_COMMS_PP_INBOUNDS` and `SECURE_COMMS_PP_OUTBOUNDS` config points. [See more details regarding Inbounds and Outbounds below.](#adding-named-tunnels-to-the-ssh-channel)
+
+Use `kubectl edit cm peer-pods-cm -n confidential-containers-system` to make such changes in the configMap, for example:
+```sh
+apiVersion: v1
+data:
+    ...
+    SECURE_COMMS: "true"
+    SECURE_COMMS_OUTBOUNDS: "KUBERNETES_PHASE:mytunnel:149.81.64.62:7777"
+    SECURE_COMMS_PP_INBOUNDS: "KUBERNETES_PHASE:mytunnel:podns:6666"
+    SECURE_COMMS_KBS_ADDR: "false"
+    ...
+```
+
```

Comment:
Some suggestions for the doc
1. Add a brief mention that this is an alternative to TLS encryption between the Adaptor and the Forwarder.
2. Add a mention about how the Attestation phase becomes optional if KBS is not used. Let's use the Attestation Phase (mandatory or optional) as a way to distinguish CoCo and Non-CoCo
3. Show two sequence of steps for SecureComms. One with Attestation Phase (and KBS). Another without the Attestation Phase. Use the same for setup instructions
4. If you can add a diagram showing a e2e example for inbound/outbound and namespaces from both the CAA and pod VM side, it'll be really helpful to visualise.